wgpu-core/src/device/mod.rs

```diff
@@ -3126,6 +3147,12 @@ pub enum CreateSwapChainError {
 pub enum CreateTextureViewError {
     #[error("not enough memory left")]
     OutOfMemory,
+    #[error(
+        "TextureView mip level count + base mip level {0} must be <= Texture mip level count {1}"
+    )]
+    InvalidMipLevelCount(u32, u8),
```

Comment:
could we have them named as `requested: u32, total: u8`, and similarly for arrays?